include/lbann/data_readers/data_reader_sample_list_impl.hpp

```diff
@@ -79,7 +79,7 @@ void data_reader_sample_list<SampleListT>::shuffle_indices(rng_gen& gen)
   // BVE FIXME get_mini_batch_size()
   auto mini_batch_size = 1;
```

Comment:
```suggestion
  auto mini_batch_size = (size_t) 1;
```

Reply:
Really it should be `decltype(get_trainer().get_max_mini_batch_size()) mini_batch_size = 1;` Isn't C++ just the best... 🤷‍♂️